admin_scripts/ubuntu/DOCUMENTATION.txt

```diff
@@ -59,8 +61,8 @@ Skifte browserstartside
 Der foretages ingen validering af, at URL'en angiver en gyldig
 hjemmeside-adresse, eller at adressen eksisterer.
 
-De to scripts kan findes som hhv. "Startside for Google Chrome" og "Startside
-for Mozilla Firefox" under "Globale Scripts".
+De to scripts kan findes som hhv. "Startside for Google Chrome" og
+"Startside for Mozilla Firefox" under "Globale Scripts".
```

Comment:
These names are no longer correct and should be `Chrome - Skift startside` and `Firefox - Skift startside` respectively.